packages/core/src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -254,6 +256,27 @@ export default class BraintreeHostedForm {
         }
     }
 
+    private _mapErrors(fields: BraintreeHostedFieldsState['fields']): BraintreeFormErrorsData {
+        const errors: BraintreeFormErrorsData = {};
+
+        if (fields) {
+            for (const [key, value] of Object.entries(fields)) {
+                if (value) {
+                    const { isValid, isEmpty, isPotentiallyValid } = value;
+                    if (this._isValidParam(key)) {
+                        errors[key] = {
+                            isValid,
+                            isEmpty,
+                            isPotentiallyValid,
+                        };
+                    }
+                }
+            }
+        }
```

Comment:
Can we simplify the code in this way?

```suggestion
        if (fields) {
            for (const [key, value] of Object.entries(fields)) {
                if (value && this._isValidParam(key)) {
                    const { isValid, isEmpty, isPotentiallyValid } = value;

                    errors[key] = {
                        isValid,
                        isEmpty,
                        isPotentiallyValid,
                    };
                }
            }
        }
```